core/libp2p/host_builder.cpp

```diff
@@ -162,10 +162,13 @@ namespace libp2p {
               io_context->get_executor());
     }
 
+    // TODO(warchant): replace with real implementation PRE-149
```

Comment:
is this still the case? Mocked implementation is commented one line below. Also please apply clang format here

Reply:
it is the case, yes. if we use mock here, we would need to link some libs, and use ugly include 
```
#include "../../../test/mock/libp2p/transport/upgrader_mock.hpp"
```